Automatic Validation ended with:
> 2024-01-25T06:52:46.9689732Z ##[error] One or more errors occurred. ((Line: 27, Col: 15, Idx: 1008) - (Line: 27, Col: 15, Idx: 1008): While scanning a multiline plain scalar, found invalid mapping.) (Line: 27, Col: 15, Idx: 1008) - (Line: 27, Col: 15, Idx: 1008): While scanning a multiline plain scalar, found invalid mapping.

(Automated response - build 655.)

--- a/manifests/v/Vivaldi/Vivaldi/6.5.3206.57/Vivaldi.Vivaldi.locale.en-US.yaml
+++ b/manifests/v/Vivaldi/Vivaldi/6.5.3206.57/Vivaldi.Vivaldi.locale.en-US.yaml
@@ -24,7 +24,7 @@ Tags:
 - internet
 - web
 - webpage
-  ReleaseNotes: |-
+ReleaseNotes: |-
   - [Calendar] Freeze when data format contains parenthesis VB-102107
   - [Chromium] Upgraded to 120.0.6099.270
   - [Linux] Update the Linux packages and repositories to use 33EAAB8E [KEY09]

